<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LocationCardEditor</class>
 <widget class="QDialog" name="LocationCardEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Место нахождения амбулаторной карты</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Место нахождения амбулаторной карты</string>
     </property>
     <property name="buddy">
      <cstring>cmbLocationCardType</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="CRBComboBox" name="cmbLocationCardType"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblPerson">
     <property name="text">
      <string>Ответственный за хранение</string>
     </property>
     <property name="buddy">
      <cstring>cmbPerson</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="CPersonComboBoxEx" name="cmbPerson"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblNotesPage">
     <property name="text">
      <string>Примечание</string>
     </property>
     <property name="buddy">
      <cstring>edtNotesPage</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1" rowspan="2">
    <widget class="QTextEdit" name="edtNotesPage"/>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>CPersonComboBoxEx</class>
   <extends>QComboBox</extends>
   <header>Orgs/PersonComboBoxEx.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cmbLocationCardType</tabstop>
  <tabstop>cmbPerson</tabstop>
  <tabstop>edtNotesPage</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>LocationCardEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel">
     <x>117</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>LocationCardEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>242</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
